### PR TITLE
Add Zlinky cluster

### PR DESCRIPTION
**Pull request by vk496**
Hello,

This is a initial patch to add support to Zlinky (https://github.com/fairecasoimeme/Zlinky_TIC).

I'm not sure if I need to modify something else, but this would be a initial step to make work the alpha integration that is being made right now (https://gist.github.com/ralmn/02de46e193bf111062f15be07d3e9bd2)

Br,
Valentin

### CODE REVIEW

**Comment by Koenkk** [comment]
Thanks, I will merge it once a PR for this device @ zigbee-herdsman-converter is made.

**Comment by vk496** [comment]
Thank you @Koenkk 

The only thing we miss is dynamically expose attributes to the user (I opened a ticket here: https://github.com/Koenkk/zigbee2mqtt/issues/9929)

Probably I will translate the attributes of this PR too....

**Comment by seblang** [comment]
Hi

À dead line to integrate this PR in official version?
Thank you

**Comment by vk496** [comment]
Hi @seblang 

Once the main is merged: https://github.com/Koenkk/zigbee-herdsman-converters/pull/3493

But for that one, https://github.com/Koenkk/zigbee2mqtt/pull/10132 is needed

**Comment by seblang** [comment]
@vk496

Thank to you and koennk for your job and your time.

I am not an expert! Have I something to do on my HA system to be able to test this new version?
Thank you and have a nice day

File: src/zcl/definition/cluster.ts [hunk 1]
```diff
@@ -4106,6 +4106,51 @@ const Cluster: {
         commands: {},
         commandsResponse: {}
     },
+    liXeePrivate: {
+        ID: 0xFF66,
```
Comment (Koenkk) [inline src/zcl/definition/cluster.ts:4110]: Please add a manufacturer code here (like https://github.com/Koenkk/zigbee-herdsman/blob/d50c3b101757cc27f19fcd4afecfb672880cc04a/src/zcl/definition/cluster.ts#L4210)

**Comment by vk496** [comment]
@Koenkk ready. It would be great to merge this PR ASAP, so people could test the zlinky integration as external converter without recompiling the whole projects together. Just installing Z2M Dev and trying

File: src/zcl/definition/cluster.ts (hunk 1 above)
Comment (vk496) [reply src/zcl/definition/cluster.ts:4207]: Just as note for people interested in finding the code: https://github.com/fairecasoimeme/Zlinky_TIC/blob/7f10743f5cb6c3c1bde4d5aaee412b99e45bfdbf/ZLinky/Build/mcux/Makefile#L146

Or get a OTA image, go to offeset 10 and look the next 2 bytes (LE encoding)

**Comment by Koenkk** [comment]
@vk496 will be in dev in an hour from now

**Comment by seblang** [comment]
Thank you both for your job.
I am not an expert! Have I something to do on my HA system to be able to test this new version?

**Pull request merged**